### ISSUE TITLE
Click on output path browse button in the build page crashes VS

### ISSUE THREAD

**Issue by srivatsn** [dotnet/roslyn-project-system#1589]
![a](https://cloud.githubusercontent.com/assets/20570/23084074/90bcb2ee-f515-11e6-9af8-59d13bf82f56.jpg)

**Comment by davkean** [comment]
```
Exception Info: System.NullReferenceException
   at Microsoft.VisualStudio.Editors.PropertyPages.CpsPropertyDescriptorWrapper..ctor(System.ComponentModel.PropertyDescriptor)
   at Microsoft.VisualStudio.Editors.PropertyPages.PropPageUserControlBase.GetPropertyDescriptor(System.String)
   at Microsoft.VisualStudio.Editors.PropertyPages.PropPageUserControlBase.GetProjectPath()
   at Microsoft.VisualStudio.Editors.PropertyPages.BuildPropPage.OutputPathBrowse_Click(System.Object, System.EventArgs)
   at System.Windows.Forms.Control.OnClick(System.EventArgs)
   at System.Windows.Forms.Button.OnClick(System.EventArgs)
   at System.Windows.Forms.Button.OnMouseUp(System.Windows.Forms.MouseEventArgs)
   at System.Windows.Forms.Control.WmMouseUp(System.Windows.Forms.Message ByRef, System.Windows.Forms.MouseButtons, Int32)
   at System.Windows.Forms.Control.WndProc(System.Windows.Forms.Message ByRef)
   at System.Windows.Forms.ButtonBase.WndProc(System.Windows.Forms.Message ByRef)
   at System.Windows.Forms.Button.WndProc(System.Windows.Forms.Message ByRef)
   at System.Windows.Forms.Control+ControlNativeWindow.OnMessage(System.Windows.Forms.Message ByRef)
   at System.Windows.Forms.Control+ControlNativeWindow.WndProc(System.Windows.Forms.Message ByRef)
   at System.Windows.Forms.NativeWindow.DebuggableCallback(IntPtr, Int32, IntPtr, IntPtr)
```

**Comment by jinujoseph** [comment]
Closed via #1590
